spring-integration-redis/src/main/java/org/springframework/integration/redis/inbound/RedisQueueInboundGateway.java

```diff
@@ -131,7 +131,7 @@ public void setReceiveTimeout(long receiveTimeout) {
 
 	/**
 	 * @param stopTimeout the timeout to block {@link #doStop()} until the last message
-	 * will be processed or this timeout is reached. Should be less then or equal to
+	 * will be processed or this timeout is reached. Should be more then or equal to
```

Comment:
`s/then/than/` (lots of places).

Also, we usually say `greater than` rather than `more than`.